### DISCUSSION TITLE
Duplicate Collection (with current fields / schema)

### DISCUSSION THREAD

**Discussion by eliteproxy7** [directus/directus#5773] [Archive]
what is the best way to duplicate a collection with all the associated fields? the content does not need to be duplicated and the new collection will be renamed

**Comment by benhaynes** [comment]
There's no way to do that as of now, mostly because relational fields and their foreign key constraints cannot necessarily be duplicated.

Love the idea though, and perhaps there's a way to add this feature without relational field duplication.
